lib/subproviders/geth_api_double.js

```diff
@@ -545,7 +545,18 @@ GethApiDouble.prototype.evm_setTime = function(date, callback) {
   callback(null, this.state.blockchain.setTime(date));
 };
 
-GethApiDouble.prototype.evm_mine = function(timestamp, callback) {
+
+GethApiDouble.prototype.evm_mineBlockNumber = function (number, callback) {
```

Comment:
Do we want to roll this into `evm_mine`?

Reply:
I am completely ambivalent on that one